app/assets/src/components/views/report/ReportTable/index.jsx

```diff
@@ -0,0 +1,153 @@
+import React from "react";
+import Tipsy from "react-tipsy";
+
```

Comment:
Remove blank lines between imports.

Is this order following  some logic of imports? Like third-party imports, idseq components and same package imports?

Reply:
Discussed in person to keep blank lines.